### PR TITLE
Add support for casting with Yle Areena app.

### PR DESCRIPTION
**Pull request by Eerovil**
https://areena.yle.fi/

This is Finlands government-run streaming service. The service uses the Kaltura player for video streaming, and thus fetching media ID's is a bit difficult. I've worked around this by just opening a page and running the javascript, to get an "Entry ID".

### CODE REVIEW

File: pychromecast/controllers/yleareena.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    # pylint: disable=useless-super-delegation
+    def __init__(self):
+        super(YleAreenaController, self).__init__()
+
+        self.app_id = APP_YLEAREENA
+
+    # pylint: disable=too-many-arguments
+    def play_areena_media(
+        self,
+        entry_id="",
```
Comment (balloob) [inline pychromecast/controllers/yleareena.py]: This is required, don't give it a default value.

**Comment by emontnemery** [comment]
@Eerovil This has been sitting for some time, do you consider it ready for merge?

**Comment by Eerovil** [comment]
I'm not sure, I'd like some input from a tester perhaps. But then again, merging won't hurt anybody. 

Personally I'm using direct media urls to cast Areena, so this feature isn't getting much use from me.

**Comment by Eerovil** [comment]
I've found a better way for fetching id's. Now I'm happy with the pull request!

**Comment by emontnemery** [comment]
Implemented in #359

**Pull request closed without merging**